src/main/frontend/components/block.cljs

```diff
@@ -762,15 +762,12 @@
 
         (= name "youtube")
         (let [url (first arguments)]
+          ;; javascript regex "^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w\-]+\?v=|embed\/|v\/)?)(?<id>[\w\-]+)(\S+)?$"
+          (def YouTube-regex #"^((?:https?:)?//)?((?:www|m).)?((?:youtube.com|youtu.be))(/(?:[\w-]+\?v=|embed/|v/)?)([\w-]+)(\S+)?$")
```

Comment:
`def` will create a global symbol in the namespace, we can use `let` here.

Reply:
done